protofsm/state_machine.go

```diff
@@ -355,11 +354,9 @@ func (s *StateMachine[Event, Env]) executeDaemonEvent(ctx context.Context,
 	// any preconditions as well as post-send events.
 	case *SendMsgEvent[Event]:
 		sendAndCleanUp := func() error {
-			log.Debugf("FSM(%v): sending message to target(%x): "+
-				"%v", s.cfg.Env.Name(),
-				daemonEvent.TargetPeer.SerializeCompressed(),
-				lnutils.SpewLogClosure(daemonEvent.Msgs),
-			)
+			log.DebugS(ctx, "Sending message",
```

Comment:
Do you have a print sample of the before and after here re formatting?

Reply:
see other comment re the prefix: with the prefixed logger, they now look like this:

```
2024-12-10 13:20:44.552 [DBG]: FSM(test) Starting state machine
2024-12-10 13:20:44.552 [DBG]: FSM(test) Sending event event=(*protofsm.emitInternal)(0x1045f1240)({
})

2024-12-10 13:20:44.552 [DBG]: FSM(test) Applying new event event=(*protofsm.emitInternal)(0x1045f1240)({
})

2024-12-10 13:20:44.552 [DBG]: FSM(test) Processing event event=(*protofsm.emitInternal)(0x1045f1240)({
})

2024-12-10 13:20:44.552 [DBG]: FSM(test) Adding new internal event to queue event !BADKEY=(*protofsm.goToFin)(0x1045f1240)({
})

2024-12-10 13:20:44.552 [INF]: FSM(test) State transition from_state=*protofsm.dummyStateStart to_state=*protofsm.dummyStateStart
2024-12-10 13:20:44.552 [DBG]: FSM(test) Processing event event=(*protofsm.goToFin)(0x1045f1240)({
})

2024-12-10 13:20:44.552 [INF]: FSM(test) State transition from_state=*protofsm.dummyStateStart to_state=*protofsm.dummyStateFin
```